diff --git a/node_modules/gulp-less/index.js b/node_modules/gulp-less/index.js
index 0d073b4..bbe6897 100644
--- a/node_modules/gulp-less/index.js
+++ b/node_modules/gulp-less/index.js
@@ -81,7 +81,7 @@ module.exports = function (options) {
     opts.filename = file.path;
 
     // Bootstrap source maps
-    if (file.sourceMap || opts.sourcemap) {
+    if ((file.sourceMap || opts.sourcemap) && !opts.sourceMap) {
       opts.sourceMap = true;
     }
 
